src/core/src/enclave_sgx.rs

```diff
@@ -77,6 +80,7 @@ pub struct EnclaveConfig {
 const DEFAULT_IAS_REPORT_KEY_FILE: &str = "src/resources/ias_report_key.pem";
 const IAS_REPORT_SIGNATURE: &str = "x-iasreport-signature";
 const MAXIMUM_NONCE_LENGTH: usize = 32;
+const TOML_FILE: &str = "src/resources/poet_config.toml";
```

Comment:
If this is used only for tests, can this be put under mod tests itself?
Similarly the use / extern crate declaration of toml.

Reply:
Ok